src/main/java/com/microsoft/azure/relay/RelayLogger.java

```diff
@@ -128,6 +128,7 @@ private static void init() {
 		map.put("httpSendResponse", "%s: Sending the response command on the %s connection, status: %s.");
 		map.put("httpSendResponseFinished", "%s: Finished sending the response command on the %s connection, status: %s.");
 		map.put("httpUserRequestHandlerException", "%s: exception in the user's request handler.");
```

Comment:
>exception in the user's request handler. [](start = 50, length = 40)

I know you're not touching this line but I happened to look at it anyway...

In the case of user request handler error do no details of the exception get logged?

If something more is needed feel free to file an issue and resolve this comment.

Reply:
Should we log any details about user implementation? I was concerned about exposing user internal details

---
In reply to: [291715446](https://github.com/Azure/azure-relay-java/pull/45#discussion_r291715446) [](ancestors = 291715446)

Reply:
It's far less common to have sensitive information in Exceptions themselves. Personally, I'd to the exception type, message, callstack in the tracing. Also, these traces go wherever our customers configure them to go after they opt-in by turning on tracing. Typically that's something that stays on the same machine as the user code. One has to be more careful when sending details over the wire or saving logs in multitenant systems or in the cloud.

---
In reply to: [292631454](https://github.com/Azure/azure-relay-java/pull/45#discussion_r292631454) [](ancestors = 292631454,291715446)